setup.cfg

```diff
@@ -31,6 +31,8 @@ zip_safe = false
 
 [options.extras_require]
 redis = 
+    redis >= 4.3.1
+aioredis =
     aioredis < 3.0.0
```

Comment:
can you also restrict here aioredis <= 2.0.0 ( I don't know how to do it exactly)